Dockerfile

```diff
@@ -11,6 +11,8 @@ RUN apt-get update && \
 
 COPY CHANGES.rst README.rst setup.py /code/
 COPY reana_server/version.py /code/reana_server/
+RUN pip install -e git://github.com/dinosk/reana-db.git@119-workflow-submission-queue#egg=reana-db
```

Comment:
We can remove this installation as in https://github.com/reanahub/reana-workflow-controller/pull/162/files#r248676641.